python/ray/serve/tests/test_air_integrations.py

```diff
@@ -90,8 +91,13 @@ def test_dataframe_with_tensorarray(self):
 
         unpacked_list = BatchingManager.split_dataframe(batched_df, 1)
         assert len(unpacked_list) == 1
-        assert unpacked_list[0]["a"].equals(split_df["a"])
-        assert unpacked_list[0]["b"].equals(split_df["b"])
+        # On windows, conversion dtype is not preserved.
```

Comment:
sad.... nice find!

Reply:
cc: @clarkzinzow for awareness .. not sure if its related to TensorArray, but in the worst case this means AIR could mess up user's tensor dtype among full / half / mixed precision training =.=